lib/screens/authentication/recover/recover_account_found/interactor/viewmodels/recover_account_found_state.dart

```diff
@@ -2,7 +2,7 @@ part of 'recover_account_found_bloc.dart';
 
 class RecoverAccountFoundState extends Equatable {
   final PageState pageState;
-  final String? errorMessage;
+  final RecoverAccountFoundError? errorMessage;
```

Comment:
A nitpick! 🤓 
`errorMessage` suggests a string. I believe `error` would be clearer. @gguijarro-c-chwy @RaulUrtecho, what do you guys think?

Reply:
error sounds good to me

Reply:
I agree too